frontend/packages/console-app/src/components/quick-starts/QuickStartCloseModal.tsx

```diff
@@ -21,17 +21,18 @@ const QuickStartCloseModal: React.FC<QuickStartCloseModalProps> = ({
       isOpen={isOpen}
       variant={ModalVariant.small}
       showClose={false}
-      title={t('console-app~Leave quick start?')}
+      data-test="leave-quickstart"
+      title={t('quickstart~Leave quick start?')}
       footer={
         <Flex>
           <FlexItem align={{ default: 'alignRight' }}>
-            <Button variant="secondary" onClick={onCancel}>
-              {t('console-app~Cancel')}
+            <Button variant="secondary" data-test="cancel button" onClick={onCancel}>
```

Comment:
these test id needs to be added in https://github.com/patternfly/patternfly-quickstarts

Reply:
Added